src/Umbraco.Web.UI/config/tinyMceConfig.config

```diff
@@ -27,6 +27,7 @@
     <command>
       <umbracoAlias>undo</umbracoAlias>
       <name>Undo</name>
+      <name>Remove Format</name>

```

Comment:
nice catch! We actually shiped with this setting in the .zip and .nuget files. It seems to take the first definition of name though so it not visible anywhere.

Ive removed the dublicate name tag now